Makefile

```diff
@@ -5,6 +5,8 @@ RM		= rm -f
 
 ifeq ($(OS),Darwin)
 	RETRACE_CFLAGS = $(CFLAGS) -fPIC -D_GNU_SOURCE -Wall
+else ifeq ($(OS),FreeBSD)
```

Comment:
No need to retype common CFLAGS over and over again, just move common part out of "if" and append what is needed for each OS.